src/pkg/packager/interactive.go

```diff
@@ -44,7 +45,28 @@ func (p *Packager) confirmAction(userMessage string, sbomViewFiles []string) (co
 
 	// Prompt the user for confirmation, on abort return false
 	if err := survey.AskOne(prompt, &confirm); err != nil {
-		return false
+		confirm = true
+	}
+
+	// On create in interactive mode, prompt for max package size
+	if userMessage == "Create" {
+		value, err := p.promptVariable(types.ZarfPackageVariable{
+			Name:        "Maximum Package Size",
+			Description: "Specify a maximum file size for this package in Megabytes (MB). Above this size, the package will be split into multiple files. 0 will disable this feature.",
+			Default:     "0",
```

Comment:
I'm sorry @jeff-mccoy ... I accidentally found this by running `zarf package create packages/big-bang-core -m 50` I expected the -m to be carried through (like we do for `components`)...

![image](https://user-images.githubusercontent.com/3977569/207387284-743e3ecd-dd91-4d91-be74-f4c05850cd2a.png)
![image](https://user-images.githubusercontent.com/3977569/207387339-81d4eb21-4ce9-4c3e-b65a-102b889522f5.png)

Can we carry forward the default through to here (either from the config toml or CLI?)